src/server.h

```diff
@@ -1828,6 +1828,8 @@ struct valkeyServer {
     long long stat_total_writes_processed;             /* Total number of write events processed */
     long long stat_client_qbuf_limit_disconnections;   /* Total number of clients reached query buf length limit */
     long long stat_client_outbuf_limit_disconnections; /* Total number of clients reached output buf length limit */
+    long long stat_total_prefetch_entries;             /* Total number of prefetched dict entries */
```

Comment:
It seems that these counters should be initialized to 0 somewhere on server start and updated in e.g. `kvstoreDictPrefetch`. Running a benchmark with a 100 clients against valkey-server with io-threads enabled shows constant value on `io_threaded_average_prefetch_batch_size`. Am I doing something wrong?